app/client/package.json

```diff
@@ -1,63 +1,64 @@
 {
-  "name": "create-react-app-typescript-redux",
-  "version": "2.0.0",
-  "dependencies": {
-    "@material-ui/core": "4.9.0",
-    "@material-ui/icons": "4.5.1",
-    "localforage": "1.7.3",
-    "react": "16.12.0",
-    "react-dom": "16.12.0",
-    "react-markdown": "^4.3.1",
-    "react-redux": "7.1.3",
-    "react-router-dom": "5.1.2",
-    "react-scripts": "3.4.1",
-    "redux-devtools-extension": "2.13.8",
-    "redux-logger": "3.0.6",
-    "redux-persist": "6.0.0",
-    "redux-thunk": "2.3.0",
-    "typescript": "3.7.5"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "cy:run": "cypress run",
-    "cyrun": "start-server-and-test start http://localhost:3000 cy:run",
-    "format": "prettier --write \"**/*.+(ts|tsx|js|jsx|json|yml|yaml|css|md|vue)\""
-  },
-  "lint-staged": {
-    "*.+(js|jsx|ts|tsx)": [
-      "prettier --write",
-      "git add"
-    ],
-    "*.+(json|yml|yaml|md|vue)": [
-      "prettier --write"
-    ],
-    "*.+(css|scss)": [
-      "prettier --write"
-    ]
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ],
-  "devDependencies": {
-    "@types/jest": "24.0.25",
-    "@types/node": "^12.12.44",
-    "@types/react": "16.9.19",
-    "@types/react-dom": "16.9.5",
-    "@types/react-redux": "7.1.7",
-    "@types/react-router-dom": "5.1.3",
-    "@types/redux-logger": "3.0.7",
-    "@types/webpack-env": "1.15.1",
-    "cypress": "^4.9.0",
-    "eslint-plugin-react": "7.17.0",
-    "husky": "^4.2.5",
-    "lint-staged": "^10.2.6",
-    "prettier": "^2.0.5",
-    "start-server-and-test": "^1.11.0"
-  }
+	"name": "create-react-app-typescript-redux",
+	"version": "2.0.0",
+	"dependencies": {
+		"@material-ui/core": "4.9.0",
```

Comment:
Why this indentation changed?

Reply:
This is happening due to the vs code configuration.

Reply:
ok! This vs code configuration, every developer doesn't have, So then they will do the change a/c to their configuration, Then how would you handle that?

Reply:
We have to add a linting setup similar to litmus portal to fix this too @chandankumar4

Reply:
I'll take that on me....after the release

Reply:
Yeah as we don't have any linting aetup here for the style to be consistent for th tab spaces i can change it to 2 from 4.

Reply:
for now you can chnage your editor to take two space for a tab and run npm run format @ishangupta-ds

Reply:
I have updated the indentation from 4 to 2 tab space.

Reply:
What about the other file change in front-end?

Reply:
@ishangupta-ds you edited the wrong file with wrong configurations...now there is 30 files change....just add one file in this PR that's the analytics.go
leave all other files

Reply:
don't put any ts, tsx, js, jsx, json files or any other frontend related files in the PR